Extensions/Physics2Behavior/tests/physics2runtimebehavior.spec.js

```diff
@@ -1,4 +1,27 @@
 describe('Physics2RuntimeBehavior', () => {
+  class FakeAutoRemoveBehavior extends gdjs.RuntimeBehavior {
+    shouldDeleteInPreEvent = false;
+
+    constructor(runtimeScene, behaviorData, owner) {
+      super(runtimeScene, behaviorData, owner);
+    }
+
+    shouldAutoRemoveInPreEvent() {
```

Comment:
nit:
`setShouldAutoRemoveInPreEvent`

if you start a method name by "shouldXXX" or "hasXXX" or "isXXX", this gives the impression this is returning a boolean. i.e: a getter rather than a setter.